core/apps/backend/account_api.py

```diff
@@ -0,0 +1,39 @@
+import allure
+from requests import Response
+
+from config.env import API_URL
+from core.apps.backend.base_api import BaseApi
+
+
+class AccountApi(BaseApi):
```

Comment:
Actually, in your case user_api and account_api are same web_api. You can split it by parts, but please create base parent api for it, set api_url there end inherit it.